static/bidder-info/adsinteractive.yaml

```diff
@@ -0,0 +1,12 @@
+endpoint: "http://bid.adsinteractive.com/prebid"
+maintainer:
+  email: it@adsinteractive.com
+gvlVendorID: 42
```

Comment:
Please either change this to your IAB GVL id, or remove if your company is not registered.